packages/connect-core/src/utils/app.ts

```diff
@@ -1,11 +1,32 @@
 import { utils as ethersUtils } from 'ethers'
 
-import { AppMethod } from '../types'
+import { Abi, AppMethod } from '../types'
 import App from '../entities/App'
 
 export const apmAppId = (appName: string): string =>
   ethersUtils.namehash(`${appName}.aragonpm.eth`)
 
+const checkSignature = (signature: string, abi: Abi): string => {
+  const regex = signature.match(/(.*)\((.*)\)/m)!
```

Comment:
Should we remove the `!` and test ourselves if the regex matches? I think we should remove any `!` remaining in the codebase as it makes the code using it unsafe.

Also should we call the variable `matches` (since the regex is on the right side here)?